31Aug_MajorityElement.py

```diff
@@ -0,0 +1,33 @@
+# https://leetcode.com/problems/majority-element/
```

Comment:
I'd probably lowercase all the names too, mixed case can cause problems with compatibility

Reply:
But this gives a better way to identify the problem statement easily.
<img width="900" alt="Screenshot 2023-09-04 at 12 42 23 PM" src="https://github.com/RevathiKannanGithub/leetcode/assets/37206283/da517393-fd3e-46ee-90c2-0e116911529e">